### PR TITLE
Fix gen levels

### PR DESCRIPTION
**Pull request by edellucien**
# How to

Search for 200 12-35, highlight the line on various zooms

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

**Comment by oterral** [comment]
Line is not good at the zoom level 7.570568967825974:
https://deploy-preview-594--dev-trafimage-maps.netlify.com/?baselayers=ch.sbb.netzkarte,ch.sbb.netzkarte.landeskarte.grau,ch.sbb.netzkarte.landeskarte,ch.sbb.netzkarte.luftbild.group&lang=de&layers=&x=696750.86&y=5877818.58&z=7.570568967825974

**Comment by edellucien** [comment]
> Line is not good at the zoom level 7.570568967825974:
> https://deploy-preview-594--dev-trafimage-maps.netlify.com/?baselayers=ch.sbb.netzkarte,ch.sbb.netzkarte.landeskarte.grau,ch.sbb.netzkarte.landeskarte,ch.sbb.netzkarte.luftbild.group&lang=de&layers=&x=696750.86&y=5877818.58&z=7.570568967825974

Good catch! I fixed the decimal zooms.

**Comment by oterral** [comment]
and on high zoom level https://deploy-preview-594--dev-trafimage-maps.netlify.com/?baselayers=ch.sbb.netzkarte,ch.sbb.netzkarte.landeskarte.grau,ch.sbb.netzkarte.landeskarte,ch.sbb.netzkarte.luftbild.group&lang=de&layers=&x=841542.09&y=5886372.11&z=16.740404407154102

but maybe it's nromal

**Comment by edellucien** [comment]
> and on high zoom level https://deploy-preview-594--dev-trafimage-maps.netlify.com/?baselayers=ch.sbb.netzkarte,ch.sbb.netzkarte.landeskarte.grau,ch.sbb.netzkarte.landeskarte,ch.sbb.netzkarte.luftbild.group&lang=de&layers=&x=841542.09&y=5886372.11&z=16.740404407154102
> 
> but maybe it's nromal

That is currently inherent from our data generalizations. High zoom isn't precise enough.

**Comment by edellucien** [comment]
> All checkboxes are not checked

Tests added and box checked :angel:

File: src/layers/layerHelper.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import layerHelper from './layerHelper';
+
+const expected = [30, 5, 100];
+
+describe('constants', () => {
+  [247.545142, 1200.999999, 40.545142].forEach((res, idx) => {
```
Comment (oterral) [inline src/layers/layerHelper.test.js]: test all the resolutions and also the default 150.

**Pull request merged**